admin_guide/quota.adoc

```diff
@@ -313,15 +334,16 @@ quota:
 ====
 ----
 $ oc describe quota object-counts -n demoproject
-Name:			object-counts
-Namespace:		demoproject
-Resource		Used	Hard
---------		----	----
-configmaps		3	10
-persistentvolumeclaims	0	4
-replicationcontrollers	3	20
-secrets			9	10
-services		2	10
+Name:                       object-counts
+Namespace:                  demoproject
+Resource                    Used    Hard
+--------                    ----    ----
+configmaps                  3       10
+openshift.io/imagestreams   4       10
```

Comment:
a little awkward that this is rendering object-counts, but this is in openshift-object-counts.  i don't think we need this line really, the goal is just to let users know generally how to describe a quota (of any name).